### ISSUE TITLE
`ubuntu-latest` is missing the .NET Core 3 SDK

### ISSUE THREAD

**Issue by dkmiller** [microsoft/azure-pipelines-image-generation#1270]
**Describe the bug**

Running a .NET Core Test with 3.0.100 on `ubuntu-latest` throws an error:

> The specified framework 'Microsoft.NETCore.App', version '3.0.0' was not found.

**Expected behavior**
The tests run successfully on .NET Core 3.0.

**Actual behavior**
Tests are _not_ running, and the build fails saying that we are missing Microsoft.NETCore.App, version 3.0.0.

1. Create a test project.
2. Use task to add .NET Core 3.0 (`netcoreapp3.0`).
3. Try running the tests on a `ubuntu-latest` image.
4. Observe the error.

**Comment by dkmiller** [comment]
This is the Ubuntu version of: #1256 .

**Comment by maxim-lobanov** [comment]
@dkmiller, as a temporary solution, you can probably install .NET Core 3.0.100 in runtime using [.NET Core Tool Installer task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops).

**Comment by dkmiller** [comment]
@maxim-lobanov yes, I am aware of that as a temporary solution. But I am hoping that the `ubuntu-latest` image will soon include that SDK by default.

**Comment by inhifistereo** [comment]
I've done what @maxim-lobanov recommended and my build is still failing. It appears the sdk installs but when I get to my Docker Build Image task I get the following error on the dotnet restore task:

"It was not possible to find any installed .NET Core SDKs‌
  Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
      https://aka.ms/dotnet-download
The command &#x27;&#x2F;bin&#x2F;sh -c dotnet restore&#x27; returned a non-zero code: 145‌
##[error]The command '/bin/sh -c dotnet restore' returned a non-zero code: 145
##[error]/usr/bin/docker failed with return code: 145"

**Comment by alepauly** [comment]
@dkmiller I'm working on it, sorry for the delay.

**Comment by dkmiller** [comment]
I'm guessing #1281 is the fix for this - thanks for the speedy turnaround. I'll be looking forward to the deployment rolling out in a few days.

**Comment by dkmiller** [comment]
@alepauly is there any ETA on deployment? Builds are still failing on `ubuntu-latest`.

**Comment by dkmiller** [comment]
Builds on `ubuntu-latest` agents targeting .NET Core 3 are finally passing, so I'm closing this out!
